tfjs-converter/src/executor/graph_model.ts

```diff
@@ -108,14 +108,26 @@ export class GraphModel implements InferenceModel {
    * Loads the model and weight files, construct the in memory weight map and
    * compile the inference graph.
    */
+
```

Comment:
since this is a new API, can you add a doc param for this method?

similar to   
/**
   * @doc {heading: 'Models', subheading: 'Classes', ignoreCI: true}
   */